### DISCUSSION TITLE
Consider adding offline support / sync

### DISCUSSION THREAD

**Discussion by Jucgshu** [audioling/audioling#8] [Ideas]
Hi,

Following [this Feishin feature request](https://github.com/jeffvli/feishin/issues/47), would it be possible to consider adding offline support?

That would help with the following:
  * Play music when connectivity is bad or non existent
  * Be able to download tracks individually or by multi selection
  * Be able to sync playlists with server

**Comment by jeffvli** [comment]
I addressed this in #2, but I do plan on adding some form of support for offline playback.

**Comment by raphmim** [comment]
Lack of offline support was the main reason I wasn't able to use feishin, so I hope this gets implemented :) I'd love to be able to make entire playlists available offline.
